cmd/kubeadm/app/cmd/phases/util.go

```diff
@@ -18,14 +18,14 @@ package phases
 
 import (
 	"k8s.io/component-base/version"
-	kubeadmapiv1beta2 "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta2"
+	kubeadmapiv1 "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3"
```

Comment:
nit: kubeadmapiv1beta3

Reply:
we discussed that we should start using `kubeadmapiv1` to reduce the diff noise (outside of the imports at least).